app/src/main/java/edu/uc/hickmadc/customnotifications/ui/main/MainFragment.kt

```diff
@@ -16,15 +16,13 @@ class MainFragment : Fragment() {
     private lateinit var viewModel: MainViewModel
 
     override fun onCreateView(
-        inflater: LayoutInflater, container: ViewGroup?,
+        inflater: LayoutInflater,
+        container: ViewGroup?,
         savedInstanceState: Bundle?
```

Comment:
made some changes with the layout of the onCreateView function to be more accessible and easier to use

Reply:
Source: https://kotlinlang.org/docs/coding-conventions.html#class-headers